tests/nlu/base/test_evaluation.py

```diff
@@ -320,6 +326,59 @@ def test_intent_evaluation_report(tmpdir_factory):
     assert result["predictions"][0] == prediction
 
 
+def test_intent_evaluation_report_large(tmpdir_factory):
+    path = tmpdir_factory.mktemp("evaluation").strpath
```

Comment:
the path comments are optional but would be great, cause we are trying to switch to `Path` now that we got rid of Python 3.5 
```suggestion
    path = tmpdir_factory.mktemp("evaluation")
```